include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -97,8 +96,10 @@ class Radiation : public ISimulationPlugin
      * frequency. Layout of the radiation array is:
      * [omega_1(theta_1),omega_2(theta_1),...,omega_N-omega(theta_1),
      *   omega_1(theta_2),omega_2(theta_2),...,omega_N-omega(theta_N-theta)]
+     * The second dimension is used to store intermediate results if command
+     * line option numJobs is used.
      */
-    GridBuffer<Amplitude, DIM1> *radiation;
+    GridBuffer<Amplitude, 2> *radiation;
```

Comment:
Why this hard coded 2?

Reply:
https://github.com/ComputationalRadiationPhysics/picongpu/pull/3354#discussion_r490109096